src/components/common/CollapsibleSection.tsx

```diff
@@ -43,17 +49,19 @@ export default function CollapsibleSection({
     }
 
     return (
-        <Details
-            open={state.sectionToggle[id] ?? defaultValue}
-            onToggle={(e) => setState(e.currentTarget.open)}
-            {...detailsProps}>
-            <summary>
-                <div class="padding">
-                    <ChevronDown size={20} />
-                    {summary}
-                </div>
-            </summary>
-            {children}
-        </Details>
+        <Wrapper noSpacing={noSpacing}>
+            <Details
```

Comment:
There could instead just be an option on `<Details />` such as `padTop` or something along those lines, instead of creating a whole new component.